### PR TITLE
Update docs 010721

### PR DESCRIPTION
**Pull request by ptrdvrk**
Updated README.md:

* Java, Ruby setup instructions in separate documents in ./doc/
* Quickstart installation section with (most of) the original short README.md content
* Images + animated gifs in ./doc/media
* Reviewed with EL on Friday

### CODE REVIEW

**Comment by kgilpin** [comment]
It looks like this needs to be rebased to master? It includes some unrelated commits from Dustin.

File: media/scenario.js [hunk 1]
```diff
@@ -51,15 +60,15 @@ import FunctionDetails from './ui/functionDetails.js';
 		scenarioData.classMap.forEach(defaultFunctionLabels);
 
 		function aggregateEvents(events, classMap) {
-			const eventInfo = new Appmap.Models.EventInfo(classMap);
-			const callTree = new Appmap.Models.CallTree(events);
```
Comment (kgilpin) [inline media/scenario.js:55]: These types of changes don't belong in this branch, right?

Comment (ptrdvrk) [reply media/scenario.js:55]: Correct. None of the .js files should be a part of this PR.

**Comment by ptrdvrk** [comment]
Will create a new branch to merge, re-basing this one is a PITA.

**Pull request closed without merging**